packages/theme-ui-expansion-pack/src/Accordion/AccordionHeader.tsx

```diff
@@ -39,7 +40,7 @@ export const AccordionHeader = forwardRef<undefined, IAccordionHeader>(
         __themeKey="accordion"
         variant="accordionHeader"
         {...rest}
-        __css={{
+        css={{
           '&[aria-expanded="true"]': {
             '[data-accoridion="icon"]': {
               transform: 'rotate(-180deg)',
```

Comment:
U ovom fileu niže, rendera se ispod `children` i svg element koji prikazuje `chevron-arrow` za accordion header...koja je onda svrha AccordionIcon komponente?

Reply:
ideja je bila da na neki nacin mozes promjenit ikonu (path). Treba to nekak osmislit. Nisam bas siguran kako. Mozda icon prop il nes slicno.

Reply:
Trebalo bi se odlučiti šta je prioritet...`AccordionIcon` ili ovaj `svg` element...ili si mislio ako ce on poslat i `iconProp` (npr.) i kao jedan od child elemenata `AccordionIcon` da se prikažu obje ikone? Što ako želi bez strelice?

Reply:
Trebali bi popisat se slucajeve i onda doc do nekog zakljucka i nabacit par prijedloga

Reply:
Uz `iconPath` prop mislim da bi svakako trebao ići još jedan prop ili barem ovaj premenovati. Pala mi je ideja, al ne znam koliko je to okej solucija. Ideja je da `iconPath` preimenujemo tipa u `withIcon` ili `indicatorIcon`ili nesto treće i on je tipa `boolean | string` te ako mu se proslijedi `true` onda on koristi defaultni path koji je sada u dokumenu `M11.5 13.7L6.1 9.2A1 1 0 005 10.8l6 5c.3.3.9.3 1.2 0l6-5A1 1 0 1017 9.2l-5.4 4.5z`, opcionalno ako netko koristi svoj icon path onda umjesto true pošalje svoj path u obliku stringa. Ili s dva propa `hasIcon` i `iconPath` npr. Mislim da bi tako dali potpunu kontrolu developeru da korisit ili neku defaulnu ikonu ili neku svoju i to sa ili bez custom stilova. Što misliš?

Reply:
napravio sam ti PR. Podsjetio sam se sto radi AccordionIcon... budes skuzio po promjenama. Isto moras mergeat master u ovaj branch.